extensions/wikia/UserLogin/UserLoginForm.class.php

```diff
@@ -246,6 +267,16 @@ public function initUser( $u, $autocreate, $skipConfirm = false ) {
 
 		wfRunHooks( 'AddNewAccount', array( $u, false ) );
 
+		/*
+		 * Remove when SOC-217 ABTest is finished
+		 */
+		if ( $isAllowRegisterUnconfirmed ) {
+			$u->setCookies();
```

Comment:
Does this just make sure the user has a login cookie during this init phase whereas they would normally only get it after their first login?